packages/stratimikos/example/tpetra/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+tribits_add_executable( simple_tpetra_stratimikos_example
+  SOURCES simple_tpetra_stratimikos_example.cpp )
+
+TRIBITS_ADD_TEST( simple_tpetra_stratimikos_example
+  EXCLUDE_IF_NOT_TRUE ${PACKAGE_NAME}_ENABLE_Amesos2
+  POSTFIX_AND_ARGS_0 amesos2_klu2
+    "--linear-solver-params-file=${CMAKE_CURRENT_SOURCE_DIR}/amesos2.klu2.xml"
+    "--matrix-file=${CMAKE_CURRENT_SOURCE_DIR}/FourByFour.mtx" --tol=1e-13
```

Comment:
Is there a reason to not copy xml inputs over to the build dir? Asking, since I haven't seen this handled this way before.

Reply:
It shortens the iterative development/test cycle.  If you make any change to an input file, you have to run the complete configure again to copy the files over.  And, technically, it avoid the potential to not copy over a file correctly with an existing build dir giving a false passing tests.

The only time I now copy files over to the build dir is when using `tribits_add_advanced_test()` where a new test subdir is created for the test to run in.  And if the test dir for that test exists when the test runs, it is deleted and re-created from scratch each time before the file is copied.  That just creates more robust tests when doing rebuilds.